### PR TITLE
Proposal: adding RESOURCES.md for examples

### PR DESCRIPTION
**Pull request by LasaleFamine**
As discussed on #329. This is my proposal.

### CODE REVIEW

**Comment by LasaleFamine** [comment]
I signed it!

File: examples/RESOURCES.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Resources and examples
+
+> Useful tools or projects as example of main Puppeteer features.
```
Comment (ebidel) [inline examples/RESOURCES.md]: "Other useful tools and projects that use Puppeteer."

File: examples/RESOURCES.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+## Web scraping
+- [Puppetron](https://github.com/cheeaun/puppetron) - Non-production (Headless Chrome Node API)-based rendering solution.
```
Comment (ebidel) [inline examples/RESOURCES.md]: Demo site that shows how to use Puppeteer and Headless Chrome to render pages. Inspired by https://github.com/GoogleChrome/rendertron.

File: examples/RESOURCES.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+ - [Thal](https://github.com/emadehsan/thal) - Getting started with Puppeteer and Chrome Headless for Web Scraping.
```
Comment (ebidel) [inline examples/RESOURCES.md]: Let's just link to the article:
https://medium.com/@e_mad_ehsan/getting-started-with-puppeteer-and-chrome-headless-for-web-scrapping-6bf5979dee3e

File: examples/RESOURCES.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+## Testing
+- [angular-puppeteer-demo](https://github.com/Quramy/angular-puppeteer-demo) - A demonstration repository explains how to replace from Karma to Puppeteer.
```
Comment (ebidel) [inline examples/RESOURCES.md]: Demo repository explaining how to use Puppeteer in Karma.

**Review by ebidel — CHANGES_REQUESTED** [review]

**Comment by LasaleFamine** [comment]
Changes made. Let me know if you need other changes. @ebidel

**Review by ebidel — APPROVED** [review]

**Pull request merged**